.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,41 @@
+name: Coding Standards
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
Please remove this line and I think it should let all upcoming `pull_request` event run the GitHub action workflow.